dts/bindings/adc/maxim,max1125x-base.yaml

```diff
@@ -0,0 +1,40 @@
+# Common fields for Texas Instruments MAX1125x
```

Comment:
Is Texas Instruments correct here?

Reply:
No, incorrect. i will replace with 'Analog Devices Inc.'